pom.xml

```diff
@@ -71,6 +72,11 @@
     		<artifactId>NetCal-NumBackend</artifactId>
 			<version>${numbackend.version}</version>
 		</dependency>
+        <dependency>
+            <groupId>org.networkcalculus.model</groupId>
+            <artifactId>NetCal-NetworkModel</artifactId>
+            <version>${networkmodel.version}</version>
+        </dependency>
```

Comment:
Indentation is different from the dependency above it which may cause the same problem with diff as you pointed out in pull request #82.